src/Conference/ConferenceDetails.jsx

```diff
@@ -44,6 +44,14 @@ const ConferenceDetails = ({ conference, id, isLoading, error }) => {
     deleteConference(id);
   };
 
+  const expressInterest = async () => {
+    try {
+      await favouriteConference(conference, token);
```

Comment:
This is fine, but I'm wondering if it would be better to call it addFavouriteConference or something - I'm assuming in the future you might also implement a deleteFavouriteConference too? Although I guess you could call it unfavouriteConference

Reply:
I think (add/delete)favouriteConference or setFavouriteConference. Depends what the functionality is. Do you actively delete a favourite or just replace it?